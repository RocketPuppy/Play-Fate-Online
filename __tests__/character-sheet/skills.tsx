import * as React from "react";
import * as renderer from "react-test-renderer";
import Skills from "../../src/character-sheet/skills";

test("renders all the pieces", () => {
  const render = renderer.create(<Skills />);

  expect(render.toJSON()).toMatchSnapshot();
});

