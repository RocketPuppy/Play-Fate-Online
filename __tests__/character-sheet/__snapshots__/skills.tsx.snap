// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Skills
  </h2>
  <form>
    <label>
      <span>
        Superb
      </span>
      <span>
        (
        <span>
          +
        </span>
        5
        )
      </span>
      <form>
        <input
          defaultValue=""
          onChange={[Function]}
          placeholder="Athletics"
          type="text"
        />
        <button
          disabled={true}
          onClick={[Function]}
        >
          Add Skill
        </button>
      </form>
    </label>
    <label>
      <span>
        Great
      </span>
      <span>
        (
        <span>
          +
        </span>
        4
        )
      </span>
      <form>
        <input
          defaultValue=""
          onChange={[Function]}
          placeholder="Athletics"
          type="text"
        />
        <button
          disabled={true}
          onClick={[Function]}
        >
          Add Skill
        </button>
      </form>
    </label>
    <label>
      <span>
        Good
      </span>
      <span>
        (
        <span>
          +
        </span>
        3
        )
      </span>
      <form>
        <input
          defaultValue=""
          onChange={[Function]}
          placeholder="Athletics"
          type="text"
        />
        <button
          disabled={true}
          onClick={[Function]}
        >
          Add Skill
        </button>
      </form>
    </label>
    <label>
      <span>
        Fair
      </span>
      <span>
        (
        <span>
          +
        </span>
        2
        )
      </span>
      <form>
        <input
          defaultValue=""
          onChange={[Function]}
          placeholder="Athletics"
          type="text"
        />
        <button
          disabled={true}
          onClick={[Function]}
        >
          Add Skill
        </button>
      </form>
    </label>
    <label>
      <span>
        Average
      </span>
      <span>
        (
        <span>
          +
        </span>
        1
        )
      </span>
      <form>
        <input
          defaultValue=""
          onChange={[Function]}
          placeholder="Athletics"
          type="text"
        />
        <button
          disabled={true}
          onClick={[Function]}
        >
          Add Skill
        </button>
      </form>
    </label>
  </form>
</div>
`;
